### DISCUSSION TITLE
Where can I see public LLM benchmarks using Vanna?

### DISCUSSION THREAD

**Discussion by andreped** [vanna-ai/vanna#578] [Q&A]
@zainhoda I remember seeing benchmark results for text2sql using Vanna with different LLMs. That was very interesting for our teams, as we were interested in exploring a subset of LLMs to try for our applications.

I believe these benchmarks were performed on a local dataset developed by Vanna.

Is there a place where I can see the current benchmark results that the Vanna team have observed?

I believe I saw some posts about this on LinkedIn at least :]

In advance, thanks!

**Comment by iiitmahesh** [comment]
@andreped @zainhoda  on update on this ? Any suggestions ?
